book/src/bridge/extern_rustqt.md

```diff
@@ -30,6 +30,13 @@ A bridge module may contain zero or more `extern "RustQt"` blocks.
 This complements the [`extern "Rust"` CXX section](https://cxx.rs/extern-rust.html)
 but allows for declaring Qt specific features on C++ types.
 
+## Automatic case conversion
+
+Automatically converting to camel or snake case can be done through the `#[auto_cxx_name]` and `#[auto_rust_name]` attributes.
+These attributes are placed on an `extern` block, and apply to all elements inside it, provided they don't specify rust or cxx names.
+`#[auto_cxx_name]` will convert the rust identifier to camelCase and use that for the cxx name.
+`#[auto_rust_name]` will convert the rust identifier to snake_case and use that for the rust name.
```

Comment:
then as we are merging sections, do you mention that you can do `#[auto_cxx_name = Snake]` as well i wonder, which could allow for further cases in the future. Or do we ignore that ? :thinking:

Reply:
Support is there, but it's kinda superfluous for now I feel so I left it out. I just can't see any case where you would need to convert to the other case than is done by default, but leon suggested leaving that door open so I implemented it for future use.

Reply:
Right, i need `ScreamingSnake` everywhere :-D